README.md

```diff
@@ -1,5 +1,6 @@
 # Agate Components Library
-An experimental Enact theme, implementing a new radical approach to what could be done with the existing skinning system.
+
+> An experimental Enact theme, implementing a new radical approach to what could be done with the existing skinning system.
```

Comment:
Should we update this while we're at it to reflect its role in auto?

Reply:
It seemed outside the scope of this ticket.  I agree we should update the README before we release this.

Reply:
ok